iohk/UpdateProposal.hs

```diff
@@ -270,33 +276,47 @@ instance Checkable UpdateProposalConfig1 where
     | otherwise = Nothing
 
 instance Checkable UpdateProposalConfig2 where
-  checkConfig (UpdateProposalConfig2 p r) = checkConfig p <|> check r
+  checkConfig (UpdateProposalConfig2 p r b s) = checkConfig p <|> check r
+                                                <|> checkInstallerHashes b
+                                                <|> checkInstallerHashes s
     where
       check InstallersResults{..}
         | grApplicationVersion <= 0 = Just "Application version must be set"
         | T.null grCardanoCommit = Just "Missing cardano commit"
         | missingVersion Mac64 ciResults = Just "macOS version is missing"
         | missingVersion Win64 ciResults = Just "Windows version is missing"
+        | missingVersion Linux64 ciResults = Just "Linux version is missing"
```

Comment:
won't this require a linux version to be passed (instead of it being optional)?

Reply:
I was thinking that because Linux installers have been reliably built in buildkite for some time, and that Linux will be fully supported in 1.4.0, there is no need to make it optional.

Does that sound alright, or should I add a flag or something?